Library/Formula/arx.rb

```diff
@@ -0,0 +1,26 @@
+class Arx < Formula
+  desc "Bundles files and programs for easy transfer and repeatable execution"
+  homepage "https://github.com/solidsnack/arx"
+  url "https://github.com/solidsnack/arx/archive/0.2.1.tar.gz"
+  sha256 "c3e91e4b9f72353061b0a104ee752da88770d695ef6329eca0f1049a56e9c110"
+
+  depends_on "ghc" => :build
+  depends_on "cabal-install" => :build
+
+  def install
+    system "cabal", "sandbox", "init"
+    system "cabal", "update"
+    system "cabal", "install", "--only-dependencies"
+    system "make"
+    bin.install "tmp/dist/arx-0.2.1-amd64-osx/arx" => "arx"
```

Comment:
Will this always be `amd64`? Do GHC/Cabal not support the old 32-bit platforms? (Can't remember off the top of my head).